security/fma-holocene.md

```diff
@@ -95,7 +103,48 @@ We would need to fix the the implementation via a hardfork. As a hotfix, we woul
 
 ##  Configurable EIP-1559 Parameters via SystemConfig
 ##  L2ToL1MessagePasser Storage Root in Header
+
 ##  Update to the MIPS contract
+
+As part of the Holocene change, the Go compiler was updated from 1.21 to 1.22. This impacts the op-program as the go1.22 runtime makes additional syscalls that are not supported by the pre-Holocene MIPS Fault Proof Virtual Machine (FPVM).
+As such, Holocene includes an update to MIPS.sol that supports go1.22 programs. This change to the FPVM is very minimal; an update to the `fcntl` syscall emulation that was partially implemented by the pre-Holocene FPVM.
+
+### Undetected go1.22 changes
+
+If there other changes were introduced by go1.22 beyond `fcntl`, and the MIPS FPVM does not emulate them correctly, it could result in blocks that are not fault-provable. This ultimately results in fault dispute games resolving incorrectly.
```

Comment:
```suggestion
If there were other changes introduced by go1.22 beyond `fcntl`, and the MIPS FPVM does not emulate them correctly, it could result in blocks that are not fault-provable. This ultimately results in fault dispute games resolving incorrectly.
```